namerd/storage/in-memory/src/test/scala/io/buoyant/namerd/storage/InMemoryDtabStoreTest.scala

```diff
@@ -86,4 +86,14 @@ class InMemoryDtabStoreTest extends FunSuite {
       extractDtab(store.observe("test")) == Dtab.read("/hello => /world")
     )
   }
+
+  test("observe composed") {
+    val store = mkStore
+    val obs = store.observeComposed("test/child")
+    assert(obs.sample.isEmpty)
+    Await.result(store.create("test/child", Dtab.read("/nice => /to/meet/you")))
+    assert(obs.sample.get.dtab == Dtab.read("/foo => /bar; /nice => /to/meet/you"))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/ver/linkerd/file/2415533482/issues/source?bid=3236810&fileBranchId=3236829#l95)